docs/source/extending_mypy.rst

```diff
@@ -59,8 +59,11 @@ new first class kinds of types.
    The plugin system is experimental and prone to change. If you want to write
    a mypy plugin, we recommend you start by contacting the mypy core developers
    on `gitter <https://gitter.im/python/typing>`_. In particular, there are
-   no guarantees about backwards compatibility. Backwards incompatible changes
-   may be made without a deprecation period.
+   no guarantees about backwards compatibility.
+
+   Backwards incompatible changes may be made without a deprecation period,
+   but we will attempt to announce them in
```

Comment:
I think we should just say "we will announce" (and actually do this :-))

Reply:
"WP:WEASEL" :P